.github/workflows/build.yaml

```diff
@@ -26,7 +26,7 @@ on:
 env:
   GO_VERSION: "1.15"
   KIND_VERSION: "v0.9.0"
-  KIND_IMAGE: "kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb"
+  KIND_IMAGE: "kindest/node:v1.19.7@sha256:a70639454e97a4b733f9d9b67e12c01f6b0297449d5b9cbbef87473458e26dca"
```

Comment:
The below test case was failing in the CI with v1.18.8, but passing locally:
Command get vulnerabilities
 for deployment/nginx resource
  vulnerabilities are associated with the deployment itself
   vulnerabilities are associated with the managed replicaset
    getting vulnerabilities by pod name

I am unable to create a local kind cluster using the v1.18.8 version, so I can't reproduce the failure. I am not sure why the kind version would have an influence on whether or not the test passes, but using the v1.19.7 version, the test passes both locally and in the CI.

Reply:
That's interesting. Have you tried creating a Kind cluster with a custom config where you could specify the version of Kubernetes?

```yaml
# three node (two workers) cluster config
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
  image: kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb
- role: worker
  image: kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb
- role: worker
  image: kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb
```

Reply:
Yeah, when I try with that version of the `kindest/node` image locally, I get:

```
ERROR: failed to create cluster: failed to init node with kubeadm: command "docker exec --privileged kind-control-plane kubeadm init --skip-phases=preflight --config=/kind/kubeadm.conf --skip-token-print --v=6" failed with error: exit status 1
```